.github/workflows/ci.yml

```diff
@@ -20,7 +20,6 @@ jobs:
       node-version: 12
       package-manager: yarn
       ember-try-scenarios: "[
-          'ember-lts-3.20',
```

Comment:
I vote to keep 3.20 if the tests are still passing, it's not a so old LTS IMO.
(Same for README and ember-try files)

Reply:
Updated following our discussion on discord, to keep compatibility and tests for last 3 LTS if possible 🎉